webpack/webpack.js

```diff
@@ -56,34 +56,43 @@ module.exports = env => ({
             utility: paths.src + '/utility/',
             react: paths.node_modules + '/react/',
             webrix: env.production ? paths.node_modules + '/webrix/' : paths.webrix,
-        }
+        },
+        fallback: {
+            assert: require.resolve('assert/'),
+            buffer: require.resolve('buffer/'),
+            fs: false,
+            module: false,
+            net: false,
+            path: require.resolve('path-browserify'),
+            process: require.resolve('process/browser'),
+        },
     },
     module: {
         rules: [
             {
                 test: /\.(js|jsx)$/,
                 exclude: /node_modules/,
                 use: {
-                    loader: "babel-loader"
-                }
+                    loader: 'babel-loader',
+                },
             },
             {
                 test: /\.s?css$/,
                 use: [
-                    "style-loader",
-                    "css-loader",
-                    "sass-loader",
+                    'style-loader',
+                    'css-loader',
+                    'sass-loader',
                     {
-                        loader: "sass-resources-loader",
+                        loader: 'sass-resources-loader',
                         options: {
                             resources: [
                                 paths.resources + '/styles/variables.scss',
                                 paths.resources + '/styles/colors.scss',
                                 paths.resources + '/styles/mixins.scss',
-                            ]
-                        }
-                    }
-                ]
+                            ],
+                        },
+                    },
+                ],
```

Comment:
I did not change all these myself, maybe the linter, not sure. Maybe it was like this already when I copied dolev's branch